app/(voters)/live-elections/page.tsx

```
@@ -0,0 +1,100 @@
+          city: entry.city,
+          state: entry.state,
+          positions: Array.from(entry.positions),
+        }));
+
+        const sorted = grouped.sort((a, b) => {
+          const placeA = `${a.city || ""}, ${a.state}`.toLowerCase();
+          const placeB = `${b.city || ""}, ${b.state}`.toLowerCase();
+          return placeA.localeCompare(placeB);
+        });
+
+        setElections(sorted as any); // cast to Election[] to match state type
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)